test/e2e/create_test.go

```diff
@@ -151,6 +159,27 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 	})
 }
 
+// setPowerShellDefaultShell changes the instance backed by the given Machine to have a default SSH shell of PowerShell
+func (tc *testContext) setPowerShellDefaultShell(machine *mapi.Machine) error {
+	// This needs to be retried as it will only succeed once the VM is fully initialized
+	// This should not increase the overall test time, as WMCO would have to wait for the VM to be ready anyway.
+	return wait.Poll(nodeRetryInterval, time.Minute*10, func() (done bool, err error) {
```

Comment:
Curious if there's a constant laying around we can use instead of just the `time.Minute*10` value?